### DISCUSSION TITLE
Some webhooks are not sent

### DISCUSSION THREAD

**Discussion by Ryokx** [postalserver/postal#2033] [Help with using Postal]
Hello,

I have set up the webhook that communicates with Mailwizz via the link: https://domain.com/index.php/dswh/postal

With the following events: MessageDelayed, MessageDeliveryFailed, MessageHeld, MessageBounced

A test was also done by choosing the global option "Yes - send all events to this URL".

The problem is the following, when a campaign is sent I want to send all bounces to Mailwizz however say that 600 messages have bounced, postal will send only 400 to Mailwizz

What could be the reason of this bug?
Or can we see the logs of Postal about the bugs of webhooks?

Thanks in advance for your help!

**Comment by willpower232** [comment]
When you say Postal stopped sending bounces to mailwizz, can you see in Postal what happened to them? The webhooks history page should tell you if there were any problems.

**Reply by Ryokx** [reply]
Postal doesn't indicate any problem from what I can see in the history page, the only detail I could notice while digging in the MYSQL database is that some webhooks queries have the "attempt" at more than 1

**Comment by Ryokx** [comment]
After analyzing the problem again, I realized that the problem came mainly from Postal, by doing a total reset of the data of Postal, I realized that on all the bases which have for example 2000 bounces the webhook will take into account only for example 1300 bounces (Figure which one can see in the page "History of the webhook"), would you have an idea or a solution please?
